recipes/xplainable/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "xplainable" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xplainable-{{ version }}.tar.gz
+  sha256: dd37e6bda3e65a0d9a6f9ecc481dd626795587f0fd0a55f8587a7a7431c90bc8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - hyperopt >=0.2.7
+    - numba >=0.56.4
+    - numpy >=1.20.3,<=1.23.5
+    - pandas >=1.5.2,<=1.9.0
+    - pyperclip >=1.8.2
+    - scikit-learn >=1.2.2
+    - scipy >=1.6.2
+    - urllib3 >=1.26.5
+    - psutil >=5.9.4
+    - joblib >=1.2.0
+    - networkx >=2.6.3
+    - tqdm >=4.62.3
+    - ipython
+    - packaging
+    - requests
+    - matplotlib >=3.4.2
```

The base variant should be sufficient.
```suggestion
    - matplotlib-base >=3.4.2
```